frontend/src/components/Analysis/AnalysisTable/index.tsx

```diff
@@ -4,17 +4,26 @@ import { ColumnsType } from "antd/lib/table";
 import { AnalysisCase, SystemOutput } from "../../../clients/openapi";
 import { backendClient, parseBackendError } from "../../../clients";
 import { PageState } from "../../../utils";
+import { taskTable } from "../AnalysisTable/supportedTasks";
+import { joinResults, addPredictionColInfo, unnestSystemOutput } from "./utils";
+
+export const seqLabTasks = [
+  "named-entity-recognition",
+  "chunking",
+  "word-segmentation",
+];
 
 interface Props {
-  systemID: string;
+  systemIDs: string[];
+  systemNames: string[];
   task: string;
   cases: AnalysisCase[];
   changeState: (newState: PageState) => void;
 }
 
 function renderColInfo(
   columns: ColumnsType<SystemOutput>,
-  colInfo: { [key: string]: string | undefined }[]
+  col_info: { [key: string]: string | undefined }[]
```

Comment:
We generally prefer camelCase instead of snake_case due to JavaScript conventions. Could we stick with `colInfo` (unless there's a reason to change)?